### DISCUSSION TITLE
Swoole security risk

### DISCUSSION THREAD

**Discussion by amir-khoshbakht** [laravel/framework#41888] [Q&A]
I see official laravel documentation recommends using swoole
[https://laravel.com/docs/9.x/octane#swoole](url)

but there are major security concern about it:
[https://news-web.php.net/php.pecl.dev/17446](url)

is it safe to use swoole or even open-swoole?

**Comment by base-zero** [comment]
This was addressed back in 2021 and the issue resolved, which a quick Google search would have showed. So based on your question being specifically about this event from October 2021 and that being resolved it's safe to say that both libraries are safe to use for this specific issue.
